Scripts/Runtime/Core/ScriptableObjectCollection.cs

```diff
@@ -398,12 +429,19 @@ public void RefreshCollection()
 
                 if (result == 0)
                 {
+                    try {
```

Comment:
Putting the brace on a new line would be more consistent with the rest of the package.
```suggestion
                    try
                    {
```